runtime/j9vm/jvm.c

```diff
@@ -1802,7 +1629,7 @@ jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args) {
 	UDATA argEncoding = ARG_ENCODING_DEFAULT;
 	UDATA altJavaHomeSpecified = 0; /* not used on non-Windows */
 	J9PortLibraryVersion portLibraryVersion;
-	UDATA j2seVersion;
+	UDATA j2seVersion = J2SE_LATEST;
```

Comment:
j2seVersion isn't really needed since you removed most of the references, you just need to fix the last one.